.vuepress/config.js

```diff
@@ -1,6 +1,6 @@
 const ogprefix = 'og: http://ogp.me/ns#'
 module.exports = {
-  title: 'MeiliSearch Documentation v0.15',
+  title: 'MeiliSearch Documentation v0.16',
```

Comment:
```suggestion
  title: 'MeiliSearch Documentation v0.18',
```
Saves you a guaranteed merge conflict ;)